### DISCUSSION TITLE
💻 WSL Pynecone Walkthrough

### DISCUSSION THREAD

**Discussion by Alek99** [reflex-dev/reflex#128] [General]
Big thanks to the @codingthunder for taking the time to make a walkthrough on how to use WSL for Pynecone!

We should have Pynecone windows compatibility stable in the coming weeks but this is a great temporary solution for windows users.

It can be found here: https://www.patreon.com/posts/76017903

**Comment by FHU-yezi** [comment]
Good article, some points to mention:

- You needn't to install Node.js and npm, all dependencies will automaticlly installed when you install Pynecone.
- Pip and Python are already installed in Ubuntu, even apt requires them.

**Reply by codingthunder** [reply]
Thanks for the feedback! I've made some quick edits to the article.

I think I ran into some issues with NPM when I was getting stuff working, but I'm actually on vacation at the moment and working off my wife's potato laptop, so I won't be able to determine if it's a replicable issue until I get back to my normal PC. For now, I'll leave some info in there in case others run into a similar problem.

Thanks again!
